project-base/config/routes/shopsys_front.yml

```diff
@@ -186,3 +186,13 @@ front_download_uploaded_file:
     methods: [GET]
     requirements:
         uploadedFileId: \d+
+
+front_customer_delivery_address_delete:
+    path: /customer/delete-delivery-address/{deliveryAddressId}
+    defaults:
+        _controller: App\Controller\Front\CustomerController:deleteDeliveryAddressAction
+        deliveryAddressId: 0
```

Comment:
why are you using default deliveryAddressId?

Reply:
fixed